src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,27 @@
+/* ==================================================
+Authour: Shane Pinto & Lucas Jacobs
+Description: Constants.java contains most of the 
+constant values used throughout the robot. This 
+includes settings, hardware ports, and other values.
+================================================== */
+
+package frc.robot;
+
+public final class Constants
+{
+    // HARDWARE PORTS
+   
+
+    // CONTROLLER COMPUTER PORTS
+    public static final int DUALSHOCK = 0;
+
+    // LIMIT SWITCH PORTS
+
+
+    // SETTINGS
+    public static final double DEADZONE = 0.15; // Deadzone applied to joysticks to aid in adjusting sensitivity
+    public static final double MAX_THROTTLE_SPEED = 0.30;
+    public static final double MAX_STEER_SPEED = 0.2;
```

Comment:
You can actually just put these constants inside `Driver.java` and `Operator.java` right at the top of the classes.  This year we decided to get rid of `Constants.java` and sort the constants into which ever file they are used in.  Keeps it more organized this way :)